cnf-certification-test/identifiers/identifiers.go

```diff
@@ -173,6 +173,12 @@ The label value is not important, only its presence.`,
 		NoDocumentedProcess,
 		bestPracticeDocV1dot3URL+" Section 5.2",
 		true,
+		claim.CategoryClassification{
+			FarEdge:  "Mandatory",
```

Comment:
should'n claim.CategoryClassification just be a map[string] string like:
type CategoryClassification struct {
   map[string] string
}
also, we could define FarEdge, Telco, NonTelco, Extended (if using a map) as constants, same with Optional, Mandatory

Reply:
Not a bad idea, gives a lot of flexibility if we need to add additional scenarios in the future.